README.md

```diff
@@ -7,3 +7,4 @@ Also, one of them had a very, very BIG secret; she could turn into a cupcake if
 Anyways, back to the story…
 They were both going shopping one day then one of them got lost.
 They also forgot where they parked their shopping cart with all of their merchandise in it.
+The shopping cart had very inportent things including; a pink tank-top, some microwavable food, and magnets (all merchandise).
```

Comment:
I love that we're being very descriptive about the items in the cart. I think there might be a typo with the spelling of "important".

```suggestion
The shopping cart had very important things including; a pink tank-top, some microwavable food, and magnets (all merchandise).
```